### PR TITLE
NMS-12078: Add support for sending events from a Minion

### PR DESCRIPTION
**Pull request by cgorantla**
Add Event Dispatcher on Minion side.
Enable event sink consumer on OpenNMS by default
Add karaf shell command to send event on Minion/OpenNMS

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12078
* Bamboo (Continuous Integration): https://bamboo.opennms.org/

### CODE REVIEW

**Comment by cgorantla** [comment]
Think I should modify EventSinkIT to send events from minion. Will do that.

**Comment by mvrueden** [comment]
AFAIK we are no longer migrating the smoke tests, but try to get the old ones to work in circlci as well instead. @j-white can you confirm this?

If we are still migrating, I suggest introducing a `@Migrated` annotation.
Then we can add that to each test class or test method which already has been migrated.

File: container/features/src/main/resources/features.xml [hunk 1]
```diff
@@ -588,6 +588,13 @@
         <feature>guava</feature>
         <bundle>mvn:org.opennms.features.events/org.opennms.features.events.commands/${project.version}</bundle>
     </feature>
+    <!--This feature lives both on minion and OpenNMS is the reason it needs separate feature.-->
+    <feature name="opennms-send-event-command" version="${project.version}" description="OpenNMS :: Events :: Send :: Command">
+        <feature>opennms-events-api</feature>
+        <bundle>mvn:org.opennms/opennms-model/${project.version}</bundle>
```
[the remark this answers is not in the document]
Comment (cgorantla) [reply container/features/src/main/resources/features.xml]: yeah,   `opennms-model`  feature has too many bundles. Don't want to load all of that.

[the remark this answers is not in the document]
Comment (j-white) [reply container/features/src/main/resources/features.xml]: Isn't it already being loaded though?

Problem is that not all of the dependencies are satisfied if we only load this feature as-is.

[the remark this answers is not in the document]
Comment (cgorantla) [reply container/features/src/main/resources/features.xml]: yeah, you are right. `org.opennms.netmgt.model.events` package seems to be there in `custom.properties`. So there is no need for `opennms-model` bundle here.

**Pull request merged**